### DISCUSSION TITLE
any way to support multiple photos for single product (part)?

### DISCUSSION THREAD

**Discussion by movelikeriver** [inventree/InvenTree#7125] [Q&A] [answered]
any way to support multiple photos for single product (part)?
currently, the image_url only support single photo

**Comment by matmair** [comment]
Yes, you can attach as many photos as you need through the attachments panel. The image_url is meant for 1 image and used to render previes etc.

**Reply by movelikeriver** [reply]
thank you!  that works

**Comment by wolflu05** [comment, marked as answer]
If you don't want to upload them as attachments, note image attachments would also work. Just drag an image into the part notes and have the pictures rendered in the markdown there.

**Reply by movelikeriver** [reply]
thank you! that's very helpful!
